code/core/src/manager/components/sidebar/RefBlocks.tsx

```diff
@@ -92,7 +91,7 @@ export const AuthBlock: FC<{ loginUrl: string; id: string }> = ({ loginUrl, id }
             <Text>Sign in to browse this Storybook.</Text>
             <div>
               {/* @ts-expect-error (non strict) */}
```

Comment:
What is non-strict about this?

Reply:
I don't know. I haven't touched this part. I can take a look, though

Reply:
![image](https://github.com/user-attachments/assets/fcf199c3-574b-4209-a996-73c420fb9820)